src/wallet.rs

```diff
@@ -271,29 +297,73 @@ struct CreateWalletParams {
 
 #[derive(Serialize, Debug, Clone)]
 struct TransferParams {
+    // Transfer from this account.
     account_index: u32,
+    // Destinations to receive XMR:
     destinations: Vec<Destination>,
+    // Set a priority for the transaction. Accepted Values are: 0-3 for: default, unimportant,
+    // normal, elevated, priority.
+    priority: u32,
+    // Number of outputs from the blockchain to mix with (0 means no mixing).
+    mixin: u32,
+    // Number of outputs to mix in the transaction (this output + N decoys from the blockchain).
+    ring_size: u32,
+    //  Number of blocks before the monero can be spent (0 to not add a lock).
+    unlock_time: u32,
+    // Return the transaction key after sending.
```

Comment:
It looks like the default values in the `new` function have a specific meaning. Since those fields are optional, can we leave them out and let monero-wallet-rpc decide what's best?

Reply:
These ones are not default according to the doc (assuming I'm reading it correctly). Priority could be interpreted as defaulting to 0 though.


https://web.getmonero.org/resources/developer-guides/wallet-rpc.html#transfer

Reply:
I meant that the hardcoded values below look like they were chosen consciously. Imho they don't matter for our usecase and we can just let the monero-walelt chose for us.

Reply:
Ok, tried and tested. You are right, even though the docs do not list these as optional the wallet does indeed pick sane values for us if they are left out. Have removed them in this PR now.